airbyte-workers/src/main/resources/dbt_transformation_entrypoint.sh

```diff
@@ -2,48 +2,54 @@
 set -e
 
 CWD=$(pwd)
-if [[ -f "${CWD}/git_repo/dbt_project.yml" ]]; then
-  # Find profile name used in the custom dbt project:
-  PROFILE_NAME=$(grep -e "profile:" < "${CWD}/git_repo/dbt_project.yml" | sed -E "s/profile: *['\"]?([^'\"]*)['\"]?/\1/")
-  if [[ -n "${PROFILE_NAME}" ]]; then
-    mv "${CWD}/profiles.yml" "${CWD}/profiles.txt"
-    # Refer to the appropriate profile name in the profiles.yml file
-    echo "Regenerate profiles.yml file for profile: $PROFILE_NAME"
-    sed -E "s/normalize:/$PROFILE_NAME:/" < "${CWD}/profiles.txt" > "${CWD}/profiles.yml"
-    rm -f "${CWD}/profiles.txt"
-    if [[ -f "${CWD}/bq_keyfile.json" ]]; then
-      cp "${CWD}/bq_keyfile.json" /tmp/bq_keyfile.json
-    fi
-  fi
-else
-  echo "git repo does not contain a dbt_project.yml file?"
-fi
+# change directory to be inside git_repo that was just cloned
+cd git_repo
+
+POSITIONAL=()
 # Detect if some mandatory dbt flags were already passed as arguments
 CONTAINS_PROFILES_DIR="false"
 CONTAINS_PROJECT_DIR="false"
-for var in "$@"; do
-    if [[ -n $(echo "${var}" | grep -e "--profiles-dir=") ]]; then
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    --profiles-dir=*|--profiles-dir)
       CONTAINS_PROFILES_DIR="true"
-    fi
-    if [[ -n $(echo "${var}" | grep -e "--project-dir=") ]]; then
+      POSITIONAL+=("$1")
+      shift
+      ;;
+    --project-dir=*|--project-dir)
```

Comment:
Confusing users that were using `--project-dir <dir>` instead of `--project-dir=<dir>` the script now handles both, see https://airbytehq.slack.com/archives/C021JANJ6TY/p1627511090382200?thread_ts=1627506369.378200&cid=C021JANJ6TY